addons/purchase/models/purchase_order.py

```diff
@@ -386,7 +386,7 @@ def _notify_by_email_prepare_rendering_context(self, message, msg_vals, model_de
         # don't show price on RFQ mail
         if self.state not in ['draft', 'sent']:
             if self.date_order:
-                subtitles.append(_('%(amount)s due\N{NO-BREAK SPACE}%(date)s',
+                subtitles.append(_('%(amount)s, Order\N{NO-BREAK SPACE}due\N{NO-BREAK SPACE}%(date)s',
```

Comment:
Wouldn't it be less confusing to simply not show the date_order on confirmed PO email? Since date_order is a deadline for the confirmation of the RFQ and it has already been confirmed, that date has no meaning anymore.

Reply:
makes sense to me